ckanext/scheming/tests/test_validation.py

```diff
@@ -287,10 +282,237 @@ def test_datetime_field_valid_separate_time(self):
 
     def test_datetime_field_in_resource(self):
         lc = LocalCKAN()
-        lc.action.package_create(type='camel-photos', name='derf_datetime', resources=[{
+        lc.action.package_create(
+            type='camel-photos',
+            name='derf_datetime',
+            resources=[{
                 'url': "http://example.com/camel.txt",
                 'camels_in_photo': 2,
-                'datetime': '2015-01-01T12:35:00'}])
+                'datetime': '2015-01-01T12:35:00'
+            }]
+        )
+
+
+class TestDateTimesTZ(object):
+    def test_datetime_field_rejects_non_isodates(self):
+        lc = LocalCKAN()
+        try:
+            lc.action.package_create(
+                type='camel-photos',
+                name='fred_datetime_tz1',
+                a_relevant_datetime_tz='31/11/2014',
+            )
+        except ValidationError as e:
+            assert_equals(e.error_dict['a_relevant_datetime_tz'],
+                          ['Date format incorrect'])
+        else:
+            raise AssertionError('ValidationError not raised')
+
+        try:
```

Comment:
Oh, I'm sorry, your validator depends on having access to the whole data_dict. Carry on (just separate out the separate tests)